src/GraphMetadataViewExtension/GraphMetadataViewExtension.cs

```diff
@@ -1,28 +1,29 @@
 ﻿using System;
 using System.Windows.Controls;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using Dynamo.Extensions;
 using Dynamo.Graph;
 using Dynamo.GraphMetadata.Properties;
 using Dynamo.Wpf.Extensions;
 using System.Windows;
 
+
 namespace Dynamo.GraphMetadata
 {
-    public class GraphMetadataViewExtension : IViewExtension, IExtensionStorageAccess
+    public class GraphMetadataViewExtension : ViewExtensionBase, IViewExtension, IExtensionStorageAccess
```

Comment:
As you are now using `ViewExtensionBase` (which is good!), we dont need `IViewExtension` as that is implemented by `ViewExtensionBase`